packages/aggregator/src/analytics/aggregateAnalytics/aggregations/sumPreviousPerPeriod.js

```diff
@@ -9,8 +9,13 @@ import { convertToPeriod, EARLIEST_DATA_DATE_STRING, getPeriodsInRange } from '@
 import { getContinuousPeriodsForAnalytics } from './utils';
 
 export const sumPreviousPerPeriod = (analytics, aggregationConfig, aggregationPeriod) => {
-  const { requestedPeriod } = aggregationConfig;
-  const periods = calculatePeriodsFromAnalytics(analytics, aggregationPeriod, requestedPeriod);
+  const { requestedPeriod, sumTillLatest } = aggregationConfig;
+  const periods = calculatePeriodsFromAnalytics(
+    analytics,
+    aggregationPeriod,
+    requestedPeriod,
+    sumTillLatest,
```

Comment:
```suggestion
    sumTillLatestData,
```
I think including the word `data` in this variable helps make it clearer that we are talking about the periods available in the data, rather than in the request period range 👍